__tests__/app.test.js

```diff
@@ -8,6 +8,8 @@ const testData = require('../db/data/test-data/index');
 const seed = require('../db/seeds/seed');
 // database connection pool for ending connection after tests run
 const db = require('../db/connection');
+// add jesr-sorted to use toBeJestSorted
+require('jest-sorted');
```

Comment:
This doesn't need to be required, it should be in your package.json jest config.